source/ub.tex

```diff
@@ -617,15 +726,28 @@
   // 2,147,483,647 Assuming 32-bit float and 64-bit double
   double d = (double)std::numeric_limits<int>::max() + 1;
   int x1 = d;   // undefined behavior 2,147,483,647 + 1 is not representable as int
-
-  __uint128_t x2 = -1;
-  float f = x2; // undefined behavior on systems where the range of
-                // representable values of float is [-max,+max] on system where
-                // represetable values are [-inf,+inf] this would not be UB
 }
 \end{codeblock}
 \end{example}
 
+\pnum
+\ubxref{conv.fpint.float.not.represented} \\
+When converting a value of integer or unscoped enumeration type to a
+floating-point type, if the value is not representable in the destination type
+it is undefined behavior.
+
+\pnum
+\begin{example}
+\begin{codeblock}
+    int main() {
```

Comment:
These next two examples are over indented, was that on purpose?

Reply:
Hopefully not. Please fix.

Reply:
Fixed.